src/ssh_audit/printconfig.py

```diff
@@ -0,0 +1,375 @@
+                PrintConfig.rocky_9_server()
+                sys.exit(retval)
+            # Ubuntu
+            elif os_type in ["Ubuntu"] and os_ver in ["2404"]:
+                PrintConfig.server_modern_common()
+                PrintConfig.ubuntu_server_2404()
+                PrintConfig.debian_ubuntu_rate_throttling()
+                sys.exit(retval)
+            elif os_type in ["Ubuntu"] and os_ver in ["2204"]:
+                PrintConfig.server_modern_common()
+                PrintConfig.ubuntu_server_2204()
+                PrintConfig.debian_ubuntu_rate_throttling()
+                sys.exit(retval)
+            elif os_type in ["Ubuntu"] and os_ver in ["2004"]:
+                PrintConfig.server_modern_common()
+                PrintConfig.ubuntu_server_2004()
+                PrintConfig.debian_ubuntu_rate_throttling()
+                sys.exit(retval)
+            elif os_type in ["Ubuntu"] and os_ver in ["1804"]:
+                PrintConfig.server_legacy_common()
+                PrintConfig.ubuntu_server_1804()
+                sys.exit(retval)
+            else:
+                PrintConfig.unknown_varient(os_type, os_ver, clientserver)
+                sys.exit(retval)
+
+
+        # Client Configs
+        if clientserver in ["Client"]:
+            # Amazon
+            if os_type in ["Amazon"] and os_ver in ["2023"]:
+                PrintConfig.amazon_2023_client()
+                sys.exit(retval)
+            # Debian
+            elif os_type in ["Debian"] and os_ver in ["Bookworm"]:
+                PrintConfig.debian_bookworm_client()
+                sys.exit(retval)
+            # Mint
+            elif os_type in ["Mint"] and os_ver in ["22"]:
+                PrintConfig.ubuntu_2404_mint_22_client()
+                sys.exit(retval)
+            elif os_type in ["Mint"] and os_ver in ["21"]:
+                PrintConfig.ubuntu_2204_mint_21_client()
+                sys.exit(retval)
+            elif os_type in ["Mint"] and os_ver in ["20"]:
+                PrintConfig.ubuntu_2004_mint_20_client()
+                sys.exit(retval)
+            # Rocky
+            elif os_type in ["Rocky"] and os_ver in ["9"]:
+                PrintConfig.rocky_9_client()
+                sys.exit(retval)
+            # Ubuntu
+            elif os_type in ["Ubuntu"] and os_ver in ["2404"]:
+                PrintConfig.ubuntu_2404_mint_22_client()
+                sys.exit(retval)
+            elif os_type in ["Ubuntu"] and os_ver in ["2204"]:
+                PrintConfig.ubuntu_2204_mint_21_client()
+                sys.exit(retval)
+            elif os_type in ["Ubuntu"] and os_ver in ["2004"]:
+                PrintConfig.ubuntu_2004_mint_20_client()
+                sys.exit(retval)
+            else:
+                PrintConfig.unknown_varient(os_type, os_ver, clientserver)
+                sys.exit(retval)
+
+
+
+    @staticmethod
+    def unknown_varient(os_type: str, os_ver: str, clientserver: str) -> None:
```

Comment:
Variant is spelled with an "a".  ;)

Reply:
I speak two languages. English and bad English.